runtime/nvqir/cutensornet/mps_simulation_state.cpp

```diff
@@ -465,9 +459,10 @@ MPSSimulationState::createFromSizeAndPtr(std::size_t size, void *ptr,
     MPSTensor stateTensor;
     stateTensor.deviceData = d_tensor;
     stateTensor.extents = std::vector<int64_t>{2};
-
-    return std::make_unique<MPSSimulationState>(
-        std::move(state), std::vector<MPSTensor>{stateTensor}, m_cutnHandle);
+    auto state =
+        TensorNetState::createFromMpsTensors({stateTensor}, cutnHandle);
+    return std::make_pair(std::move(state),
+                          std::vector<MPSTensor>{stateTensor});
```

Comment:
```suggestion
    return {std::move(state),
                          std::vector<MPSTensor>{stateTensor}};
```
The new school syntax.